flormcalvo/poc-csyd-mentor
# DEV-006/Routing links to pages

## Rationale

Added routing links to pages:
- Home page - list of mentors when visiting main home link or click on the Code Sydney Icon
- Single Mentor page - when clicking on one of the mentor's cards
- New Mentor Form page - when clicking on the New Mentor icon

Also updated the Navbar with icon to Add a Mentor (links to Mentor form)

## Advice for Reviewers & Testing Notes

1. npm install in the client directory as new npm package 'react-router-dom' has been added
2. run client / server locally
3. check out the functionality of the links, and views esp mobile view.

As you can see there's a massive warning in the console... haven't figured how to fix that haha, leaving it for now. And implementing Material-ui is a bit messier than I originally thought haha...

## Screenshots:

![dev-006-routing](https://user-images.githubusercontent.com/7751526/79689890-0abe3200-829b-11ea-9c9f-f007d36dae87.gif)


## Task Name

- Task name from Notion: DEV-006 Routing 


## Linting Checklist
The following is a list of linting rules that ESLint is not currently linting for. Please make sure your code conforms to this list. Examples/definitions of the rules can be found [here](https://hireup.atlassian.net/wiki/pages/viewpageattachments.action?pageId=618037301&metadataLink=true&preview=/618037301/618168365/PR_Checklist.pdf)
